src/constants/useAlertMessages.ts

```diff
@@ -126,6 +126,18 @@ export const useAlertMessages = () => {
       body: i18n.t('emergencyNotification.body'),
       primaryButton: i18n.t('emergencyNotification.primaryButton'),
       secondaryButton: i18n.t('emergencyNotification.secondaryButton')
+    },
+    connectElderlySuccess: {
+      header: i18n.t('connectElderlySuccess.header'),
+      body: i18n.t('connectElderlySuccess.body'),
+      primaryButton: i18n.t('connectElderlySuccess.primaryButton'),
+      secondaryButton: i18n.t('connectElderlySuccess.secondaryButton')
+    },
+    connectElderlyError: {
+      header: i18n.t('connectElderlyError.header'),
+      body: i18n.t('connectElderlyError.body'),
+      primaryButton: i18n.t('connectElderlyError.primaryButton'),
+      secondaryButton: i18n.t('connectElderlyError.secondaryButton')
```

Comment:
This is my bad, but I forgot to change from `i18n.t` to `t` in my previous PR. Please`t` from `useTranslation` instead.

Up to you if you want to implement this naa. If not, let me know I can fix it in a future PR.